package/yast2-users.changes

```diff
@@ -1,4 +1,44 @@
 -------------------------------------------------------------------
+Tue Mar  7 13:49:49 UTC 2017 - jreidinger@suse.com
+
+- do not ask again for already approved weak password second
+  attempt, so it will work also if widget is recreated(bsc#1025835)
+- 3.1.57.7
```

Comment:
previous one in this branch already have 3.1.57.1, so I keep it consistent. To be honest I do not care as long as we increase version.